.travis.yml

```diff
@@ -4,8 +4,6 @@ python:
 
 cache:
   directories:
-    - schema_editor/node_modules
```

Comment:
Any ideas for avoiding this, @ddohler @kshepard? Travis is trying to use its cache, which conflicts with the symlinks. I haven't been able to find anything that allows me to run commands before the cache gets applied to delete the symlinks.